python/producers/hhh6bProducerPNetAK4.py

```diff
@@ -874,13 +879,13 @@ def selectLeptons(self, event):
             el.Id = el.charge * (-11)
             #if el.pt > 35 and abs(el.eta) <= 2.5 and el.miniPFRelIso_all <= 0.2 and el.cutBased:
             if self.Run==2:
-                if el.pt > 10 and abs(el.eta) <= 2.5 and abs(el.dxy) < 0.045 and abs(el.dz) < 0.2 and el.miniPFRelIso_all <= 0.2 and el.lostHits <= 1 and el.convVeto and el.mvaFall17V2noIso_WP90: #and el.cutBased>3: # cutBased ID: (0:fail, 1:veto, 2:loose, 3:medium, 4:tight)
+                if el.pt > 15 and abs(el.eta) <= 2.5 and abs(el.dxy) < 0.045 and abs(el.dz) < 0.2 and el.pfRelIso03_all <= 0.2 and el.lostHits <= 1 and el.convVeto and el.mvaFall17V2Iso_WP90: #and el.cutBased>3: # cutBased ID: (0:fail, 1:veto, 2:loose, 3:medium, 4:tight)
```

Comment:
In AN 23 151 it says el.miniPFRelIso_all <= 0.15 and geometrical cuts are eta dependent. The EG twiki does not have anything explicit, and checking other random AN's with leptons, they are not consistent. I would stay consistent with AN 23 151. That one is run3 though... The run2 resolved one does not have anything on electrons ([AN2019_250_v6.pdf](http://cms.cern.ch/iCMS/jsp/openfile.jsp?tp=draft&files=AN2019_250_v6.pdf)

Reply:
Fixed using AN2019-250-v6, cut is 0.15 there. I updated to match

Reply:
thanks! will now update our AN accordingly